groovy/globals/Petrochemistry.groovy

```diff
@@ -158,14 +158,15 @@ class Petrochemistry {
     ]
 
     public static crackables = [
-        vacuum_oil_residue : new Crackable('vacuum_oil_residue').withTraits(CatalyticCrackable, HydroCrackable, ThermalCrackable).tap { hydrogen_consumed = 3560; gas_produced = 550 },
-        atmospheric_oil_residue : new Crackable('atmospheric_oil_residue').withTraits(CatalyticCrackable, HydroCrackable, ThermalCrackable).tap { hydrogen_consumed = 3540; gas_produced = 610 },
-        light_gas_oil : new Crackable('light_gas_oil').withTraits(HydroCrackable, SteamCrackable).tap { hydrogen_consumed = 880; gas_produced = 1790 },
-        heavy_gas_oil : new Crackable('heavy_gas_oil').withTraits(CatalyticCrackable, HydroCrackable, SteamCrackable).tap { hydrogen_consumed = 1490; gas_produced = 1020 },
+        vacuum_oil_residue : new Crackable('vacuum_oil_residue').withTraits(CatalyticCrackable, HydroCrackable, ThermalCrackable).tap { hydrogen_consumed = 15680; gas_produced = 550 },
```

Comment:
What resource did you find implying that hydrogen cracking was more inefficient in terms of hydrogen than you initially thought?

Reply:
i was reading my calculations spreadsheet and realized i was doing it wrong